features/0793-unqualfied-dids-transition/README.md

```diff
@@ -0,0 +1,102 @@
+# Aries RFC 0793: Unqualified DID Transition
+
+- Authors: [Sam Curren](mailto:swcurran@cloudcompass.ca)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2023-07-11
+- Status Note: In Pre-work **Target Completion Date: 2023.10.15** 
+- Supersedes:
+- Start Date: 2023-07-11
+- Tags: [feature](/tags.md#feature), [community-update](/tags.md#community-update)
+
+## Summary
+
+Historically, Aries use of the Indy SDK's wallet included the use of 'unqualified DIDs' or DIDs without a did: prefix and method. 
+This transition documents the process of migrating any such DIDs still in use to fully qualified DIDs.
+
+The transition from unqualified DIDs to peer:did:2 is described here: https://github.com/TimoGlastra/legacy-did-transformation
```

Comment:
I think there are a couple of open issues on this, and the README itself has open questions.  What about merging that into this document as an appendix, instead of in a Timo’s account?

Reply:
Agreed!